storage/kvstore_test.go

```diff
@@ -108,14 +108,14 @@ func TestStorePut(t *testing.T) {
 		wact := []testutil.Action{
 			{"put", []interface{}{keyBucketName, tt.wkey, data}},
 		}
-		if g := b.tx.Action(); !reflect.DeepEqual(g, wact) {
+		if g := s.b.(*fakeBackend).tx.Action(); !reflect.DeepEqual(g, wact) {
```

Comment:
let's assign the fackbackend to a variable at the very beginning. Same for the index.

```go
fackBackend := s.b(*fackBackend)
```